tests/e2e/visual/targets/targets-overview.wdio-spec.js

```diff
@@ -0,0 +1,694 @@
+    await pregnancyVisitForm.submitDefaultPregnancyVisit();
+    await commonPage.waitForPageLoaded();
+    await commonPage.openFastActionReport('pregnancy_home_visit');
+    await pregnancyVisitForm.submitDefaultPregnancyVisit();
+
+    await commonPage.openFastActionReport('delivery');
+
+    await commonEnketoPage.selectRadioButton('What is the outcome for the woman?', 'Alive and well');
+    await genericForm.nextPage();
+    await dangerSignPage.selectAllDangerSignsDelivery('No');
+    await genericForm.nextPage();
+    await commonEnketoPage.selectRadioButton('How many babies were delivered?', '1');
+    await commonEnketoPage.selectRadioButton('How many babies are alive?', '1');
+    await commonEnketoPage.selectRadioButton('Where did delivery take place?', 'Home');
+    await commonEnketoPage.selectRadioButton('Who conducted the delivery?', 'Skilled health care provider');
+    await commonEnketoPage.setDateValue('Date of delivery', BABY_DOB);
+    await genericForm.nextPage();
+    await commonEnketoPage.selectRadioButton('What is the condition of baby?', 'Alive and well');
+    await commonEnketoPage.setInputValue('Name', BABY_NAME_6);
+    await commonEnketoPage.selectRadioButton('Sex', 'Male');
+    await commonEnketoPage.selectRadioButton('Birth weight', 'I don\'t know');
+    await commonEnketoPage.selectRadioButton('Birth length', 'I don\'t know');
+    await commonEnketoPage.selectRadioButton('What vaccines have they received?', 'None');
+    await commonEnketoPage.selectRadioButton('Is the child exclusively breastfeeding?', 'Yes');
+    await commonEnketoPage.selectRadioButton('Were they initiated on breastfeeding within on hour of delivery?', 'Yes');
+    await commonEnketoPage.selectRadioButton('Infected umbilical cord', 'No');
+    await commonEnketoPage.selectRadioButton('Convulsions', 'No');
+    await commonEnketoPage.selectRadioButton('Difficulty feeding or drinking', 'No');
+    await commonEnketoPage.selectRadioButton('Vomits everything', 'No');
+    await commonEnketoPage.selectRadioButton('Drowsy or unconscious', 'No');
+    await commonEnketoPage.selectRadioButton('Body stiffness', 'No');
+    await commonEnketoPage.selectRadioButton('Yellow skin color', 'No');
+    await commonEnketoPage.selectRadioButton('Fever', 'No');
+    await commonEnketoPage.selectRadioButton('Blue skin color (hypothermia)', 'No');
+    await genericForm.nextPage();
+    await genericForm.nextPage();
+    await commonEnketoPage.selectCheckBox('Which PNC visits have taken place so far?', 'None of the above');
+    await genericForm.nextPage();
+
+    const summaryTexts = [
+      pregnantWoman17.name,
+      '25', //age
+      'Alive and well', //woman's condition
+      BABY_DOB,
+      'Home' //delivery place
+    ];
+
+    await commonEnketoPage.validateSummaryReport(summaryTexts);
+    await genericForm.submitForm();
+  
+  });
+  it('New Pregnancy form creation and delivery form(without facility)', async () => {
```

Comment:
My idea of implementing this test to simplify the code, is not using the UI to generate the data.
The structure could follow this pattern:
```
describe('...', () => {
  // Generate test data (people, pregnancies, deliveries) using functions already available

  before(async () => {
    // Save generated data
    // Create user
    // Log in
  });

  it('should go to Targets and take the screenshots', async () => {
    // Go to Targets
    // Take the screenshots
  });
});
```
This keeps data setup and test actions clearly separated and easier to maintain.